### DISCUSSION TITLE
[Application] Estuary

### DISCUSSION THREAD

**Discussion by jimmylee** [filecoin-project/community#105] [Project Showcase]
Learn about https://estuary.tech, built by Applications Research Group (ARG), here.

> Estuary is a network node that bridges IPFS ⇆ Filecoin. Unlike other nodes, Estuary ensures your storage and retrieval goals match your own storage heuristics. Estuary uses automation to make Filecoin storage deals, restore IPFS pins with Filecoin retrieval deals, and repairs deals against reliable miners. The outcome is that there is a higher guarantee of on chain data, and a higher guarantee your data won't disappear from the internet.

Team:
@whyrusleeping @jimmylee

**Comment by jimmylee** [comment]
## Previews

![estuary-tech](https://user-images.githubusercontent.com/310223/117113226-07c79f80-ad3f-11eb-9ae6-0f3a41a2651e.png)

https://user-images.githubusercontent.com/310223/117113285-1ada6f80-ad3f-11eb-9ba1-f5018d0c631d.mp4

<img width="1803" alt="Screen Shot 2021-05-02 at 9 43 41 PM" src="https://user-images.githubusercontent.com/310223/117114091-3e51ea00-ad40-11eb-8d13-3c7db8887f87.png">

## Updates

For the week of May 5th, 2021 — Happy Cinco de Mayo! Today, we're getting ready for an early prototype preview.

* Documentation is coming soon!
* Thank you @ognots, @travisperson and @coryschwartz for the help.
* https://estuary.tech can be shared, you just need an invite to use it, ask @whyrusleeping.
* The Miners that Estuary uses are shown on the front page, thank you miners for helping us make storage deals.

<img width="2154" alt="Screen Shot 2021-05-05 at 1 24 45 AM" src="https://user-images.githubusercontent.com/310223/117114436-b7514180-ad40-11eb-9714-8fc35e73a36f.png">

## Links

* **Estuary WWW**
  * The front-end if you wanted to run your own Estuary website speaking to your own Estuary node.
  * We're still building our design system and website out in the open! Come help :-)
  * https://github.com/application-research/estuary-www
* **Estuary Node**
  * This is where the magic happens.
  * Documentation coming soon if you want to run your own Estuary!
  * **API Source Code**: https://github.com/application-research/estuary/blob/master/handlers.go
  * https://github.com/application-research/estuary
* **Social**
  * https://twitter.com/wwwjim/status/1389860045500809217?s=20

**Reply by ztxdbo** [reply]
I need an invitation code as a storage test if I want to move my site to Estuary

**Reply by jimmylee** [reply]
What is the content of your website?

**Reply by ztxdbo** [reply]
> What is the content of your website?
Do some movies and TV shows

**Reply by greentornado** [reply]
Can I get an invitation too?

**Reply by web3jenks** [reply]
Hi @greentornado you can actually request for an invite here: https://docs.estuary.tech/get-invite-key
